doc/developers/minimal_reproducer.rst

```diff
@@ -199,21 +199,21 @@ As already mentioned, the key to communication is the readability of the code
 and good formatting can really be a plus. Notice that in the previous snippet
 we:
 
-    - try to limit all lines to a maximum of 79 characters to avoid horizontal
-      scrollbars in the code snippets blocks rendered on the GitHub issue;
-    - use blank lines to separate groups of related functions;
-    - place all the imports in their own group at the beginning.
+- try to limit all lines to a maximum of 79 characters to avoid horizontal
+    scrollbars in the code snippets blocks rendered on the GitHub issue;
+- use blank lines to separate groups of related functions;
+- place all the imports in their own group at the beginning.
 
 The simplification steps presented in this guide can be implemented in a
 different order than the progression we have shown here. The important points
 are:
 
-    - a minimal reproducer should be runnable by a simple copy-and-paste in a
-      python terminal;
-    - it should be simplified as much as possible by removing any code steps
-      that are not strictly needed to reproducing the original problem;
-    - it should ideally only rely on a minimal dataset generated on-the-fly by
-      running the code instead of relying on external data, if possible.
+- a minimal reproducer should be runnable by a simple copy-and-paste in a
+    python terminal;
+- it should be simplified as much as possible by removing any code steps
+    that are not strictly needed to reproducing the original problem;
+- it should ideally only rely on a minimal dataset generated on-the-fly by
+    running the code instead of relying on external data, if possible.
```

Comment:
```suggestion
- a minimal reproducer should be runnable by a simple copy-and-paste in a
  python terminal;
- it should be simplified as much as possible by removing any code steps
  that are not strictly needed to reproducing the original problem;
- it should ideally only rely on a minimal dataset generated on-the-fly by
  running the code instead of relying on external data, if possible.
```